src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -78,7 +78,9 @@ class ChaptersButton extends TextTrackButton {
     let items = this.items = [];
 
     for (let i = 0, length = tracks.length; i < length; i++) {
-      let track = tracks[i];
+
+      // We will always choose the last track as our chaptersTrack
+      let track = tracks[length - 1];
```

Comment:
Ok, will try to see if I can move things around. :)

Regarding choosing the last track, earlier it was like this

```
for loop {
let track = tracks[i] 
(if track matches kind as chapters)  {
  chaptersTrack = track;
  break;
 }
}

Now let's say i = 1 and kind matches with chapters, 
but we want the 2nd track( i = 2) to be the correct and most recent 
chapters Track that has to be added to chapters list
```

This will cause the `chaptersTrack` to always take value of the first track, and break out from the loop.
If this happens, the last most recent track added never gets added to the list as `chaptersTrack` will always take value of the first previous track. Would this be not the case ?